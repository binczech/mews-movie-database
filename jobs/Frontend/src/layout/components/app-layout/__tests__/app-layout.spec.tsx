import { render, RenderResult } from '@testing-library/react';
import React from 'react';

import { AppLayout } from '..';

describe('AppLayout', () => {
  const setup = (): RenderResult => render(
    <AppLayout>
      <div>Some child</div>
    </AppLayout>,
  );
  it('should match the snapshot', () => {
    expect(setup().container).toMatchSnapshot();
  });
});
