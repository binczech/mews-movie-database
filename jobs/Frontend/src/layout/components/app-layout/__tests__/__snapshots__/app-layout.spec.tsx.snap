// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppLayout should match the snapshot 1`] = `
<div>
  <section
    class="ant-layout"
  >
    <header
      class="ant-layout-header"
    >
      <div
        class="sc-bcXHqe zUhEm"
      >
        <h3
          class="ant-typography"
        >
          Device Checker
        </h3>
      </div>
    </header>
    <main
      class="ant-layout-content"
    >
      <div>
        Some child
      </div>
    </main>
  </section>
</div>
`;
